.changeset/config.json

```diff
@@ -7,5 +7,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": ["@monokle/cli"]
```

Comment:
We need automated tests for a sanity test or this will keep breaking.

Something as simple as importing a file without a `.js` postfix will do the trick already and it's hard to catch for both editors / manual reviews. Not even the build of the CLI will fail, only during runtime it will crash.